scripts/github-fetch-pullrequest

```diff
@@ -11,6 +11,8 @@ Usage:
         repository
 """
 
+from __future__ import print_function
```

Comment:
is this needed? it's enough to have script in Py3 only

Reply:
Ok, I was not sure if you require python 2 for scripts since this was python 2 only. I'll remove it.